### PR TITLE
fix: Updating the node environment to Node.js 20

### PR DESCRIPTION
**Pull request by iennae**
## Description

updates some region tags, updates node environment

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [X] I have followed guidelines from [CONTRIBUTING.MD](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md) and [Samples Style Guide](https://googlecloudplatform.github.io/samples-style-guide/)
- [] **Tests** pass:   `npm test` (see [Testing](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#run-the-tests-for-a-single-sample))
- [] **Lint** pass:   `npm run lint` (see [Style](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#style))
- [X] This pull request is from a branch created directly off of `GoogleCloudPlatform/nodejs-docs-samples`. Not a fork.
- [X] Please **merge** this PR for me once it is approved

I verified that the changes worked (region tags don't impact deployment)

### CODE REVIEW

File: appengine/building-an-app/build/server.js [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 'use strict';
 
-// [START app]
```
Comment (pattishin) [inline appengine/building-an-app/build/server.js:17]: Change affects one region tag that is still active. https://github.com/GoogleCloudPlatform/nodejs-docs-samples/pull/3617/checks?check_run_id=21334110961

File: appengine/building-an-app/build/server.js [hunk 2]
```diff
@@ -14,7 +14,7 @@
```
Comment (pattishin) [inline appengine/building-an-app/build/server.js:14]: Update copyright year to this year ☝🏼  https://github.com/GoogleCloudPlatform/nodejs-docs-samples/pull/3617/checks?check_run_id=21334109681

File: appengine/building-an-app/update/server.js [hunk 3]
```diff
@@ -14,42 +14,42 @@
```
Comment (pattishin) [inline appengine/building-an-app/update/server.js:14]: Update copyright year to this year ☝🏼 too  https://github.com/GoogleCloudPlatform/nodejs-docs-samples/pull/3617/checks?check_run_id=21334109681

**Review by pattishin — CHANGES_REQUESTED** [review]
Thanks @iennae!  👍🏼  Added a few comments based from `snippet-bot` and `header-check`.

File: appengine/building-an-app/update/server.js (hunk 3 above)
Comment (iennae) [reply appengine/building-an-app/update/server.js:14]: In general, we shouldn't be updating the copyright for files is my understanding.

Comment (pattishin) [reply appengine/building-an-app/update/server.js:14]: Required of `header-check` bot if a file is changed to also update the year.

Comment (iennae) [reply appengine/building-an-app/update/server.js:14]: sorry for the long delay in responding, just checked the configuration and it seems like license year is ignored in this repo. go/copyright for policy, but in general we shouldn't be updating copyright year on updating a file (including if we move the file)

File: appengine/building-an-app/build/server.js (hunk 2 above)
Comment (iennae) [reply appengine/building-an-app/build/server.js]: The problem was with the Google Inc versus Google LLC

File: appengine/building-an-app/update/server.js (hunk 3 above)
Comment (iennae) [reply appengine/building-an-app/update/server.js]: The problem was with the Google Inc versus Google LLC

**Review by kweinmeister — APPROVED** [review]

**Pull request merged**